tox.ini

```diff
@@ -1,12 +1,11 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{37,38,39,310}-{linux,macos,windows}
-toxworkdir = /tmp/.tox
```

Comment:
would it be possible to leave `toxworkdir = /tmp/.tox` in here?  did you run into issues?

Reply:
I was just trying to make everything like the napari-cookiecutter:
https://github.com/napari/cookiecutter-napari-plugin/blob/main/%7B%7Bcookiecutter.plugin_name%7D%7D/tox.ini

So I nuked it without thinking or testing—sorry!
I'll put it back.

Reply:
yeah, really, I'd kinda prefer this line to be in the cookiecutter too.  It's a bit personal preference.  Without it, tox directories get created all over the place (in each project), and tend not to be cleaned up.  That can grow in size over time.  By putting it in /tmp it's clear that it's just temporary junk.  
That can look a bit weird on windows though, so I left it out of the cookiecutter

Reply:
Ah, makes sense.
All pass, so will mark this as resolved.